### PR TITLE
Fixup benchmarking tutorial

### PR DESCRIPTION
**Pull request by davetcoleman**
I tested the instructions just now and clarified things @kirstyellis

### CODE REVIEW

File: doc/benchmarking_tutorial.rst [hunk 1]
```diff
@@ -1,36 +1,48 @@
-Benchmarking tutorial
+Benchmarking Tutorial
 =====================
+
+.. note:: This is the new benchmarking method only available in ROS Kinetic, onward. Documentation is lacking for the previous benchmarking tutorial for Jade and earlier.
+
+.. note:: To use this benchmarking method, you will need to download and install are ROS Warehouse plugin. Currently this is not available from debians and requires a source install for at least some aspects. For source instructions, see the bottom of [this page](http://moveit.ros.org/install/source/).
```
Comment (v4hn) [inline doc/benchmarking_tutorial.rst]: typo "and install are ROS Warehouse plugin"

Comment (davetcoleman) [inline doc/benchmarking_tutorial.rst]: fixed

**Comment by davetcoleman** [comment]
ping so that ros answers user has latest tutorial: http://answers.ros.org/question/244017/problems-with-the-old-moveit-tutorials-about-benchmark-tools/?answer=244677#post-id-244677

**Comment by gavanderhoorn** [comment]
O. I should've checked.

I just submitted #16.

**Comment by gavanderhoorn** [comment]
I changed quite a few more things than @davetcoleman. Shall I integrate those changes in #16, or would the preference be for me to rebase on this as soon as it's merged?

**Comment by davetcoleman** [comment]
This is silly something as simple as this PR isn't merged in after 6 days...

@gavanderhoorn you are a maintainer and can merge things :) how about you merge this then rebase your commit on top?

**Comment by gavanderhoorn** [comment]
I know I can merge things :) I was just wondering whether there would be a preference for a specific workflow.

I'll go ahead and merge this one, then fix #16.

File: doc/benchmarking_tutorial.rst [hunk 2]
```diff
@@ -1,36 +1,48 @@
-Benchmarking tutorial
+Benchmarking Tutorial
 =====================
+
+.. note:: This is the new benchmarking method only available in ROS Kinetic, onward. Documentation is lacking for the previous benchmarking tutorial for Jade and earlier.
+
+.. note:: To use this benchmarking method, you will need to download and install the ROS Warehouse plugin. Currently this is not available from debians and requires a source install for at least some aspects. For source instructions, see the bottom of [this page](http://moveit.ros.org/install/source/).
```
Comment (gavanderhoorn) [inline doc/benchmarking_tutorial.rst]: This is not a valid `rst` link. See the `benchmarking package` link in the paragraph below for the correct formatting.

**Comment by gavanderhoorn** [comment]
There's actually a tiny formatting issue that needs fixing @davetcoleman. If you fix that, this can be merged.

File: doc/benchmarking_tutorial.rst [hunk 3]
```diff
@@ -1,36 +1,48 @@
-Benchmarking tutorial
+Benchmarking Tutorial
 =====================
+
+.. note:: This is the new benchmarking method only available in ROS Kinetic, onward. Documentation is lacking for the previous benchmarking tutorial for Jade and earlier.
+
+.. note:: To use this benchmarking method, you will need to download and install the ROS Warehouse plugin. Currently this is not available from debians and requires a source install for at least some aspects. For source instructions, see `this page <http://moveit.ros.org/install/source/dependencies/>`_
```
Comment (davetcoleman) [inline doc/benchmarking_tutorial.rst]: good catch, fixed

**Comment by davetcoleman** [comment]
I've fixed the mis-formatted link. I also had an uncommitted change in my local repo that I sloppily added to this PR:

I removed the OMPL build instructions from the tutorial page because they were recently added to  http://moveit.ros.org/install/source/dependencies/ instead

File: doc/ompl_interface_tutorial.rst [hunk 4]
```diff
@@ -28,25 +28,3 @@ The configuration of these optimization objectives can be done in the *ompl_plan
...
-
-    git clone https://github.com/ompl/ompl
-
-Next manually add a package.xml as used in the ROS release wrapper for OMPL (modify ROS distro version as necessary)
-
-.. code-block:: bash
-
-    wget https://raw.githubusercontent.com/ros-gbp/ompl-release/debian/kinetic/xenial/ompl/package.xml
-
-Now you should be able to build using regular ``catkin build``
```
Comment (gavanderhoorn) [inline doc/ompl_interface_tutorial.rst]: Were these changes part of the original PR?

**Comment by gavanderhoorn** [comment]
Ah, that explains it. I thought I'd missed something.

**Comment by gavanderhoorn** [comment]
There is now a period missing at the end of the sentence where you fixed the link, but I'll add it in #16.

**Comment by gavanderhoorn** [comment]
I'm using *rebase and merge* here btw (as proposed in ros-planning/moveit.ros.org#76).

Not sure I'd use it for actual feature or larger PRs, but seeing as these are minor changes ..

**Comment by davetcoleman** [comment]
Thanks!

**Comment by v4hn** [comment]
@gavanderhoorn rebase-and-merge and squash-and-merge are *identical* for pull-requests of one commit. For more on the topic: https://github.com/ros-planning/moveit.ros.org/pull/76#discussion_r80955517

**Pull request merged**